src/Tasks/Microsoft.NET.Build.Tasks/FrameworkPackages/FrameworkPackages.netcoreapp3.0.cs

```diff
@@ -0,0 +1,197 @@
+namespace Microsoft.ComponentDetection.Detectors.NuGet;
```

Comment:
Do we need these when they're all out-of-support?

Reply:
It doesn't hurt, it may help a bit, and it's easier to just copy all of the code from the folder in the component-detection repo rather than to pick and choose.

Reply:
Yes it helps for out of support frameworks.  Less noise that folks need to deal with -- helps them see the key message 
 -- framework is out of support.